benchmarks/django_simple/requirements.txt

```diff
@@ -0,0 +1,17 @@
+pytz==2021.1  # https://github.com/stub42/pytz
```

Comment:
We can make dependabot maintain this file for us if we want.
Do we want to upgrade the library versions here?

Reply:
My thoughts on this is that we want the dependencies here for a test to be fixed over time, the only change being the tracer we are installing.